hal/inc/hal_dynalib_wlan.h

```diff
@@ -78,6 +78,8 @@ DYNALIB_FN(27, hal_wlan, wlan_set_hostname, int(const char*, void*))
 DYNALIB_FN(28, hal_wlan, wlan_get_hostname, int(char*, size_t, void*))
 DYNALIB_FN(29, hal_wlan, wlan_connected_info, int(void*, wlan_connected_info_t*, void*))
 DYNALIB_FN(30, hal_wlan, wlan_get_antenna, WLanSelectAntenna_TypeDef(void*))
+DYNALIB_FN(31, hal_wlan, wlan_get_ipaddress, int(IPConfig*, void*))
```

Comment:
how about we combine this into a more generic get IP configuration?

Reply:
It does actually retrieve the full *saved* configuration. The idea was for this method to be analogous to `wlan_set_ipaddress, void(const HAL_IPAddress*, const HAL_IPAddress*, const HAL_IPAddress*, const HAL_IPAddress*, const HAL_IPAddress*, void*)`, but I can see how something like `wlan_get_stored_configuration()` would work better. Let me know your thoughts on naming.